Description: Expected failing tests
 There are three categories of failing tests:
 1 - because the test only fails on a buildd server.
 2 - the test itself is buggy (race condition under heavy load,
     problem with the external command that is called) and the
     cause of the failure is a timeout.
Forwarded: not-needed
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2013-08-09
--- a/test/simple/simple.status
+++ b/test/simple/simple.status
@@ -1 +1,62 @@
 prefix simple
+
+# only fail when some network resources are not available
+test-http-dns-fail: PASS || FAIL
+test-net-connect-timeout: PASS || FAIL
+test-dgram-broadcast-multi-process: PASS || FAIL
+test-net-listen-fd0: PASS || FAIL
+test-child-process-fork-dgram: PASS || FAIL
+test-cluster-dgram-1: PASS || FAIL
+test-cluster-dgram-2: PASS || FAIL
+test-dgram-bind: PASS || FAIL
+test-dgram-bind-default-address: PASS || FAIL
+test-dgram-implicit-bind: PASS || FAIL
+test-dgram-listen-after-bind: PASS || FAIL
+test-dgram-multicast-setTTL: PASS || FAIL
+test-dgram-pingpong: PASS || FAIL
+test-dgram-udp4: PASS || FAIL
+test-dgram-unref: PASS || FAIL
+test-tls-client-destroy-soon: PASS || FAIL
+
+# slow platform - plain TIMEOUT of FAIL because of timeouts
+# i manually checked those tests pass when timeouts are raised.
+test-cluster-disconnect: PASS || TIMEOUT
+test-force-repl: PASS || FAIL
+test-http-client-timeout-with-data: PASS || FAIL
+test-http-end-throw-socket-handling: PASS || FAIL
+test-http-exit-delay: PASS || FAIL
+test-net-GH-5504: PASS || FAIL
+test-net-server-unref: PASS || FAIL
+test-net-settimeout: PASS || FAIL
+test-next-tick-error-spin: PASS || FAIL
+test-pipe-unref: PASS || FAIL
+test-stream2-readable-empty-buffer-no-eof: PASS || FAIL
+test-tls-fast-writing: PASS || FAIL
+test-next-tick-starvation: PASS || FAIL
+# this one is tricky but falls nonetheless in this category
+test-timers-unref: PASS || FAIL
+
+
+
+# http://bugs.debian.org/660802
+# test-cluster-kill-workers: PASS || FAIL
+
+# 512 bits keys fail with openssl 1.0.1 - node tests will use 1024 bits
+# keys : https://github.com/joyent/node/commit/ef046bf4
+# Let it fail until the real cause if found.
+test-tls-server-verify: PASS || FAIL
+
+# debugger tests can timeout on buildd or fail because debugger prompt has changed
+test-debugger-repl-utf8: PASS || TIMEOUT || FAIL
+test-debugger-repl-break-in-module: PASS || TIMEOUT || FAIL
+test-debugger-repl: PASS || FAIL
+test-debugger-client: PASS || FAIL
+test-debugger-repl-restart: PASS || FAIL
+
+# https://buildd.debian.org/status/fetch.php?pkg=nodejs&arch=armel&ver=0.10.15~dfsg1-1&stamp=1375130627
+test-debugger-repl-break-in-module: PASS || FAIL
+
+
+# https://buildd.debian.org/status/fetch.php?pkg=nodejs&arch=i386&ver=0.10.15~dfsg1-1&stamp=1375127251
+test-child-process-fork-net2: PASS || TIMEOUT
+
